### PR TITLE
Css canvas medical

### PR DESCRIPTION
**Pull request by AnthonyUrbina**
Screen Shot 2022-08-29 at 11 22 06 PM
Screen Shot 2022-08-29 at 11 22 17 PM
Screen Shot 2022-08-29 at 11 22 19 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Anthony! This looks good, but your code is not really using layout classes, and it is important that you make proper use of them from this point forward. Please revise your solution to make use of `container`, `row` and `column-` classes appropriately. I encourage you to re-read `css-layout-classes` for a refresher on the concept, feel free to ask me any questions you might have on the concept, I am happy to help 🤓

**Comment by AnthonyUrbina** [comment]
ok, thanks for not letting me slide on this. i think i started to panic which is why I decided to proceed this way. ill take my time and do it right. thanks, will reach out if i need to

**Comment by AnthonyUrbina** [comment]
https://user-images.githubusercontent.com/109625982/188371063-c7519e93-89fb-498b-b995-8f940ad8b8c1.mov

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+          <p>About&nbsp;Us</p>
+        </div>
+        <div class="header-col-2-text">
+          <p>Blog</p>
+        </div>
+      </div>
+      <div class="header-col-3">
+        <button>Contact Us</button>
+      </div>
+    </div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:35]: Layout classes should not be named after specific sections. You should have one `container` class, one `row` class that you can apply to any row in the layout, and one variant of `column-` classes for each size, that is reused for each element that needs to be that size.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AnthonyUrbina** [comment]
https://user-images.githubusercontent.com/109625982/189503434-30d70c40-cb86-46cb-9ac3-ac9fb9bd727a.mov

**Review by codingcodymiller — APPROVED** [review]
MUCH better!!! 🎉

**Pull request merged**